### PR TITLE
Example service

### PR DESCRIPTION
**Pull request by daniel-dgi**
So here's the smallest thing I could think of that showcases what I've been trying to describe.  It's pretty small, and shows that really all we're trying to do is hack info out of the request and return responses.  This is what is meant by 'Thin Controllers'.  All the heavy lifting is done elsewhere.  In this case, it's Chullo.

As things grow, we can either have a lot of these little guys, or try and manage all of them in a single deployable project.

### CODE REVIEW

**Comment by daniel-dgi** [comment]
To run:

```bash
$ cd /path/to/islandora/services/TransactionService/src
$ php -S localhost:9999 index.php
```

File: services/TransactionService/src/index.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+use Symfony\Component\HttpFoundation\Response;
+
+date_default_timezone_set('UTC');
+
+$app = new Application();
+
+$app['debug'] = true;
+
+$app['fedora'] = function () use ($app) {
+    $client = new Client(['base_uri' => 'http://127.0.0.1:8080/fcrepo/rest']);
```
Comment (whikloj) [inline services/TransactionService/src/index.php]: Still new to this stuff, is there a reason you didn't use the static constructor with the base_uri and get back the Chullo?
ie.
```
return Chullo::create('http://127.0.0.1:8080/fcrepo/rest');
```

Comment (daniel-dgi) [inline services/TransactionService/src/index.php]: No reason in particular.  Doing it manually gives you a bit more control over the client, which in this case, is unneccessary.

File: services/TransactionService/src/index.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            $app->abort($response->getStatusCode(), $response->getReasonPhrase());
+        }
+    }
+);
+
+$app->post(
+    "/islandora/transaction/{id}/rollback",
+    function (Application $app, $id) {
+        try {
+            return $app['fedora']->rollbackTransaction($id);
```
Comment (nigelgbanks) [inline services/TransactionService/src/index.php]: Probably should remove this 'return'

**Pull request closed without merging**